SECURITY.md

```diff
@@ -40,10 +40,12 @@ As these 3rd party audits are completed and issues are sufficiently addressed, w
 - **[April 2023 - Trail of Bits](https://github.com/wormhole-foundation/wormhole-audits/blob/main/Wormhole_Audit_Report_TrailOfBits_2023-04.pdf)**: _Guardian node: Governor and Watchers_
 - **[April 2023 - OtterSec](https://github.com/wormhole-foundation/wormhole-audits/blob/main/Wormhole_OtterSec_Sui_2023-04.pdf)**: _Sui Contracts_
 - **[May 2023 - Runtime Verification](https://github.com/wormhole-foundation/wormhole-audits/blob/main/2023-05_Runtime_Verification_Wormhole_EVM.pdf)**: _Formal Verification of EVM contracts_
-- **[Jan 2024 - Cyfrin](https://github.com/wormhole-foundation/wormhole-audits/blob/main/2024-01-10-cyfrin-thermae-v2.0.pdf)**: _Native Token Transfers EVM Contracts_
-- **Jan 2024 - OtterSec (Report coming soon)**: _Terra Classic Contract Upgrades_
-- **Jan 2024 - Cyfrin (In progress)**: _CCTP EVM Contracts_
-- **Feb 2024 - (Planned)**: _CCTP Solana Contracts_
+- **[Jan 2024 - Cyfrin](https://github.com/wormhole-foundation/wormhole-audits/blob/main/2024-01-10-cyfrin-thermae-v2.0.pdf)**: _Uniswap Liquidity Layer EVM Contracts_
+- **[Jan 2024 - OtterSec](https://github.com/wormhole-foundation/wormhole-audits/blob/main/2024-01-ottersec-terra.pdf)**: _Terra Classic Contract Upgrades_
```

Comment:
file not found

Reply:
Yup I preemptively set the path to what it will be when https://github.com/wormhole-foundation/wormhole-audits/pull/9 is merged